app/helpers.py

```diff
@@ -1,4 +1,5 @@
 from flask import make_response, jsonify
+from string import punctuation
 
 def get_topics_response(topics, status_code):
```

Comment:
Perhaps we can rename these helpers since they're named get_* but the work they seem to be doing is formatting.

Reply:
I feel the naming is ok because the helpers format responses and that's why I give them name as .....response

Reply:
How about `format_topics_response` then. When you see get_x as a reader of the code you tend to assume it is a load operation being performed while no loading is being done here. A transform is being performed.